CDP4IME/CDP4IMEBootstrapper.cs

```diff
@@ -153,5 +159,28 @@ private void UpdateBootstrapperState(string message)
             logger.Debug(this.state);
             DXSplashScreen.SetState(this.state);
         }
+
+        /// <summary>
+        /// Save the application settings when new plugins are loaded
+        /// </summary>
+        private void SaveSettings(IAppSettingsService<ImeAppSettings> appSettingsService, ICollection<string> newPlugins)
+        {
+            var modules = ServiceLocator.Current.GetAllInstances(typeof(IModule));
```

Comment:
Is it possible to use  
```
ServiceLocator.Current.GetAllInstances<IModule>()
```
Then the type cast ```(IModule)module``` below won't be needed anymore.

Reply:
Cosmetic change aplied